server/views/pages/goal/archive/reason.test.ts

```diff
@@ -0,0 +1,105 @@
+import nunjucks from 'nunjucks'
+import * as cheerio from 'cheerio'
+import ArchiveGoalView from '../../../../routes/archiveGoal/archiveGoalView'
+import aValidPrisonerSummary from '../../../../testsupport/prisonerSummaryTestDataBuilder'
+import formatReasonToArchiveGoalFilter from '../../../../filters/formatReasonToArchiveGoalFilter'
+import ReasonToArchiveGoalValue from '../../../../enums/ReasonToArchiveGoalValue'
+import findErrorFilter from '../../../../filters/findErrorFilter'
+
+const njkEnv = nunjucks.configure([
+  'node_modules/govuk-frontend/dist/',
+  'node_modules/@ministryofjustice/frontend/',
+  'server/views/',
+  __dirname,
+])
+njkEnv.addFilter('formatReasonToArchiveGoal', formatReasonToArchiveGoalFilter)
+njkEnv.addFilter('findError', findErrorFilter)
+
```

Comment:
Re: my comment above about white space ... this test is hard to read 😢

Reply:
Added some whitespace, hope that helps